WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -32,9 +32,13 @@ Most of the features introduced in the Semi-Annual Channel will be rolled up int
 The Semi-Annual Channel is available to volume-licensed customers with [Software Assurance](https://www.microsoft.com/licensing/licensing-programs/software-assurance-default.aspx), as well as via the Azure Marketplace or other cloud/hosting service providers and loyalty programs such as Visual Studio Subscriptions.
 
 > [!Note]
-> **The current Semi-Annual Channel release is Windows Server, version 2004**. If you want to put servers in this channel, you should install Windows Server, version 2004, which can be installed in Server Core mode or as Nano Server run in a container. In-place upgrades from a long-term servicing channel release aren't supported because they are in **different release channels**. Semi-Annual Channel releases aren't updates – it's the next Windows Server release in the Semi-Annual Channel.
+> **The current Semi-Annual Channel release is Windows Server, version 2004**. If you want to put servers in this channel, you should install Windows Server, version 2004, which can be installed in Server Core mode or as Nano Server run in a container. In-place upgrades from a long-term servicing channel release aren't supported because they are in **different release channels**. This applies vice versa. You cannot upgrade or change from Semi-Annual Channel to a LTSC channel, without a clean installation.
+>
+> A Semi-Annual Channel release isn't an update – it's the next Windows Server release in the Semi-Annual Channel.
+> In-place upgrades from one Semi-Annual Channel release to a later Semi-Annual Channel release are possible. This makes it easier to keep up with the relatively short release cadence.
 
 In this model, Windows Server releases are identified by the year and month of release: for example, in 2017, a release in the 9th month (September) would be identified as **version 1709**. Fresh releases of Windows Server in the Semi-Annual Channel will occur twice each year. The support lifecycle for each release is 18 months.
+Starting with fall 2020 (20H2) releases, we changed the identifcation. Instead of a month, we will name the release based on the release cycle. For example: **version 20H2**, for a release in second half of the year 2020, or **version 21H1**, for a release in the first half of the year 2021.
```

Comment:
codestyle/grammar: add missing definite article. Also: typo correction.
```suggestion
Starting with fall 2020 (20H2) releases, we changed the identification. Instead of a month, we will name the release based on the release cycle. For example: **version 20H2**, for a release in the second half of the year 2020, or **version 21H1**, for a release in the first half of the year 2021.
```